pcdsdevices/at2k4_tmo.py

```diff
@@ -0,0 +1,55 @@
+"TMO sample calibration foil to work with spectrometer"
+from lightpath import LightpathState
+from ophyd.device import Component as Cpt
+
+from .device import GroupDevice
+from .device import UpdateComponent as UpCpt
+from .epics_motor import BeckhoffAxis
+from .interface import BaseInterface, LightpathMixin
+from .pmps import TwinCATStatePMPS
+
+
+class CalibrationAxe(TwinCATStatePMPS):
+    """
+    Sample calibration 1D State Setup
+
+    Here, we specify 5 states, and 1 motor, for Y
+    axe.
+    """
+    config = UpCpt(state_count=5, motor_count=1)
+
+
+class SCaliFoil(BaseInterface, GroupDevice, LightpathMixin):
+    """
+    TMO PMPS1D sample calibration class.
+
+
+    Parameters:
+    -----------
+    prefix : str
+        Base PV for the motion system
+
+    name : str
+        Alias for the device
+    """
+    # UI Representation
+    _icon = 'fa.minus-square'
+    tab_component_names = True
+
+    # Sample calibration Y axe
+    at2k4 = Cpt(CalibrationAxe, 'AT2K4:IP1:STATE', add_prefix=(), kind='normal')
+    at2k4_y = Cpt(BeckhoffAxis, ':MMS', doc="Y-axis of sample caliration at2k4", kind='normal')
+    removed = False
+    transmission = 1
+    SUB_STATE = 'state'
+
+    # dummy signal, state is always the same
+    lightpath_cpts = ['at2k4.user_readback']
+
+    def calc_lightpath_state(self, **kwargs) -> LightpathState:
```

Comment:
We should make sure this is doing the right thing for lightpath.  If there is indeed an attenuator then we probably want to keep track of the blades to determine transmission and insertion state.  Maybe something similar to [AttenuatorSXR_Ladder](https://github.com/pcdshub/pcdsdevices/blob/c1bfa9cd54b4d875d043d12fa30ed3096d16cc25/pcdsdevices/attenuator.py#L1071)?

Reply:
Thanks Robert. No, this is not real attenuator like at1k4. no blades. Just a state mover with 5 targets on beamline with PMPS.